app/geo/gui/src/ts/twxplore/gui/geo/components/map/Map.tsx

```diff
@@ -71,22 +74,21 @@ const MapImpl: React.FunctionComponent = () => {
         to display its location and shape on the map.
         */ 
         const datasets = {
-          data: {
+          data: Processors.processGeojson({
             type: "FeatureCollection",
             features: featuresInState.map(feature => {
               return {
                 type: "Feature",
-                //geometry: wkt.parse(null),
+                geometry: wkt.parse(feature.geometry.wkt),
                 properties: feature,
               };
             }),
-          },
+          }),
           info: {
-            id: featuresInState[0].uri,
-            label: "ok"
+            id: featuresInState[0].uri
```

Comment:
I'm not sure about this. I think you'd be better off having a name/counter for a query, attaching that name to the feature (that was retrieved using that query), and using that as the id?

Reply:
Don't need to change it in this PR, though.